sdk/src/genesis_config.rs

```diff
@@ -56,7 +56,7 @@ impl ClusterType {
     pub fn get_genesis_hash(&self) -> Option<Hash> {
         match self {
             Self::MainnetBeta => {
-                Some(Hash::from_str("5eykt4UsFv8P8NJdTREpY1vzqKqZKvdpKuc147dw2N9d").unwrap())
+                Some(Hash::from_str("Cmz2Q9FETPotP8oLVQeRZvGvjdaDxJ2CdTviE4xStNZt").unwrap())
```

Comment:
this PR changes the localized blockhash to statically read the latest blockhash from mainnet-internal. It's only used when activating rekeyed feature gates